docs/deployment/run-in-prod.md

```diff
@@ -0,0 +1,212 @@
+# Running in a Production Environment
+
+## Run using Process Managers
+
+### What is a Process Manager?
+
+In modern computers and operating systems, the processor helps execute multiple programs which are arranged by the process manager. 
+In other words, the **process manager** arranges the execution of the processes or programs by making schedule for to 
+ensure efficient use of the resources which includes both software and hardware.
+
+A process manager will handle the socket setup, start-up multiple server processes, monitor process aliveness, and 
+listen for signals to provide for processes restarts, shutdowns, or dialing up and down the number of running processes.
+
+
+
+### Using a process manager
+Running Uvicorn using a process manager ensures that you can run multiple processes in a resilient manner, and 
+allows you to perform server upgrades without dropping requests.
+
+
+!!! question "Do you know?"
+    Uvicorn provides a lightweight way to run multiple worker processes, for example `--workers 4`, but does not provide any process monitoring.
+
+There are several tools you can use as process manager. In this section we will see how to use the following tools:
+
+- Gunicorn
+- Supervisor
+- Circus
```

Comment:
I have seen Supervisor or Circus being used to serve web apps in production, is these something people do?

I think we can cover:

1. Built in process manager
2. Gunicorn
3. Kubernetes / Cloud Run / PaaS offerings

What do you think?

Reply:
It was there in the previous Uvicorn docs. I just copy pasted the same. I don't think we need - but that's my opinion.